src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/PercentageConverter.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class PercentageConverter : IValueConverter
```

Comment:
i personally prefer if a converter is named like XtoY converter, so it's clear what it does

Reply:
what about `ValueToPercentageConverter`?

Reply:
yeah, that seems fine. You could also use DoubleToPercentageConverter